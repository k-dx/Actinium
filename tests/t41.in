let a = [1,[20, [30, 31]],3];
a[1][1][0] = 42;
print a;

let b = [1,[20, [30, 31]],3];
b[1][1] = 42;
print b;

let c = [1,[20, [30, 31]],3];
c[1] = 42;
print c;