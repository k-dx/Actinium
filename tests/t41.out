[1, [20, [42, 31]], 3]
[1, [20, 42], 3]
[1, 42, 3]